src/s2wasm.h

```diff
@@ -669,7 +669,7 @@ class S2WasmBuilder {
       skipComma();
       auto curr = allocator.alloc<Store>();
       curr->type = type;
-      int32_t bytes = getInt();
+      int32_t bytes = getInt()/8;
```

Comment:
that's the bits per character, isn't it? on some systems the bits per character might not be 8?